jax/scipy/special.py

```diff
@@ -15,6 +15,11 @@
 # Note: import <name> as <name> is required for names to be exported.
 # See PEP 484 & https://github.com/google/jax/issues/7570
 
+# These functions are exposed in both jax.nn (for backwards compatibility), and
+# jax.scipy.special (to support the Array API specification).
```

Comment:
Is this comment correct? I don't think the Array API says anything about scipy APIs.

Reply:
You're right.  However, maybe I'm jumping the gun, but it [probably will soon enough](https://github.com/data-apis/array-api/issues/725).  They are almost surely going to add a "special functions" extension, and these functions are in the draft there.

Also, from the linked table, they appear to be in `scipy.special`, `torch.special`, and `cupyx.scipy.special`, so there's a consistency argument for having them there?

Reply:
As I read it, it proposes a `special` namespace, not a `scipy.special` namespace.

Reply:
@hawkinsp I'm assuming that if the `special` namespace is added, Jax will simply link it to `jax.scipy.special`?

Reply:
It really depends what the final specification is.

But for now, adding these functions to `jax.scipy.special` is well in-scope for JAX (see https://jax.readthedocs.io/en/latest/jep/18137-numpy-scipy-scope.html#axis-1-xla-alignment)

Reply:
Okay, got it.  I was just trying to be proactive 😄   What I'll do then is try to produce shims with the appropriate axis behaviour, right?

Reply:
Yeah, that sounds great.